src/main/java/org/folio/ebsconet/mapper/OrdersMapper.java

```diff
@@ -99,8 +105,8 @@ public void ebsconetToFolio(MappingDataHolder mappingDataHolder) {
     populateCostAndLocations(poLine, ebsconetOrderLine);
     removeZeroAmountLocations(poLine);
 
-    if (ebsconetOrderLine.getType() != null && ebsconetOrderLine.getType().equalsIgnoreCase("non-renewal")) {
```

Comment:
"non-renewal".equalsIgnoreCase(ebsconetOrderLine.getType())

It is shorter form and without possible NPE